InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/intake/IntakeSubsystem.java

```diff
@@ -33,14 +36,14 @@ public IntakeSubsystem(Config config) {
     @Override
     public void initialize() {
         super.initialize();
-        if (config.intake.intakePivotEnabled){
+        if (IntakeConfig.intakePivotEnabled){
             intakePivet = new DoubleSolenoid(RobotMap.INTAKE_PNEUMATIC_OPEN_CHANNEL, RobotMap.INTAKE_PNEUMATIC_CLOSED_CHANNEL); 
             intakePivet.set(Value.kForward);
             SmartDashboard.putString(getName() + "/Intake Pivet", "Enabled");
         } else {
             SmartDashboard.putString(getName() + "/Intake Pivet", "Disabled");
         }
-        motor = MotorUtils.makeSRX(config.intake.intake);
+        motor = MotorUtils.makeSRX(new MotorConfig());

```

Comment:
We should use the configurable motor. It doesn't have any PID constants or anything currently, but we should allow for that option in the future.